src/modules/quick_constants_module.f90

```diff
@@ -99,6 +99,19 @@ module quick_constants_module
    & 'TL','PB','BI','PO','AT','RN','FR','RA','AC','TH', &
    & 'PA','U '/)
 
+    integer, dimension(0:92) :: SPINMULT = &
+   & (/0, 2, 1, 2, 1, 2, 3, 4, 3, 2, 1, &
```

Comment:
Looks good. Thanks! But do we need a zero element? I would arrange as follows as it is easy to check:
```
 (/0, &
   2,                                                 1, &
   2, 1,                               2, 3, 4, 3, 2, 1, &
   2, 1,                               2, 3, 4, 3, 2, 1, &
   2, 1, 2, 3, 4, 7, 6, 5, 4, 3, 2, 1, 2, 3, 4, 3, 2, 1, &
   2, 1, 2, 3, 6, 7, 6, 5, 4, 1, 2, 1, 2, 3, 4, 3, 2, 1, &
   2, 1, &
     2, 1, 4, 5, 6, 7, 8, 9, 6, 5, 4, 3, 2, 1, &
         2, 3, 4, 5, 6, 5, 4, 3, 2, 1, 2, 3, 4, 3, 2, 1, &
   2, 1, 2, 3, 4, 5/)
```
I think you do not need the ampersand at the beginning of the continuation line. See e.g. here https://pages.mtu.edu/~shene/COURSES/cs201/NOTES/chap01/continue.html